src/components/Filters/index.tsx

```diff
@@ -0,0 +1,245 @@
+import React from 'react';
+import { Material } from '@balena/ui-shared-components';
+import { useTranslation } from '../../hooks/useTranslation';
+import { Search } from './Search';
+import { Views } from './Views';
+import { FiltersDialog } from './FiltersDialog';
+import {
+	type FormData,
+	createFullTextSearchFilter,
+	randomUUID,
+	FULL_TEXT_SLUG,
+	parseFilterDescription,
+} from './SchemaSieve';
+import { Summary } from './Summary';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faFilter } from '@fortawesome/free-solid-svg-icons/faFilter';
+import { useClickOutside } from '../../hooks/useClickOutside';
+import type { JSONSchema7 as JSONSchema } from 'json-schema';
+import { isJSONSchema } from '../../AutoUI/models/helpers';
+
+const { Box, Button, useTheme, useMediaQuery } = Material;
+
+export interface FiltersView {
```

Comment:
Note for future: I'd like to revisit `Views`. Saved filters makes sense as a feature, but personally I'm not sure `views` is a global term (I could be wrong)

Reply:
Huh, didn't know it was really a thing